cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -342,9 +428,9 @@ policy "policy_recommendations" do
         label "ID"
         path "resourceID"
       end
-      field "service" do
-        label "Service"
-      end
+      # field "service" do
+      #   label "Service"
+      # end
```

Comment:
Should we keep this field as a comment? (From line 431 to 433).

Reply:
Thanks @juanestrahcl , this was addressed. I removed the commented code.